Templating/Helper/LocaleHelper.php

```diff
@@ -43,7 +43,8 @@ public function language($code, $locale = null)
     {
         $codes = explode('_', $code);
 
-        $name = Intl::getLanguageBundle()->getLanguageName($codes[0], isset($codes[1]) ? $code[1] : null, $locale ?: $this->localeDetector->getLocale());
+        $name = Intl::getLanguageBundle()->getLanguageName($codes[0], isset($codes[1]) ? $codes[1] : null, 
```

Comment:
`Intl::getLanguageBundle()` is hardcoded in the test, that's why it is not possible to test if `getLanguageName` is called with the right thing…